src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -65,6 +66,34 @@ async function validate_input_types(type, action, argv) {
     }
 }
 
+/**
+ * validate_no_leading_space ensures that flags do not contain leading spaces
+ *  - if `argv_` contains more than two elements (type and action), it indicates an invalid flag value.
+ *
+ *
+ * ### CLI Parsing Behavior:
+ * When passing arguments via CLI, if a flag's value starts with a space or consists only of whitespace,
+ * the shell may treat it differently:
+ *  - Leading spaces - might be preserved when quoted (`--new_buckets_path=" abc/"`).
+ *  - Unquoted values with spaces - may cause incorrect parsing (`--new_buckets_path= abc/` is interpreted as an empty value).
+ *  - The parsed argument structure typically follows:
+ *      ```json
+ *      { _: [ "command", "action" ], flag1: "value1", flag2: "value2", ... }
+ *      ```
+ *  - The `_` array should contain only the command and action. If additional elements appear,
+ *    it suggests an issue with flag values (e.g., an incorrectly formatted or empty value).
+ *
+ * @param {string[]} argv_
+ * @throws {ManageCLIError}
+ */
+function validate_no_leading_space(argv_) {
+    // checking if 'argv_' contains more then 2 values (i.e, type and action)
+    if (argv_.length > 2) {
```

Comment:
From what I understand from the Slack discussion, checking the size can be improved.
If you need in the file `manage_nsfs_constants.js` you have the list of types and actions:
https://github.com/noobaa/noobaa-core/blob/23aec8dcd195c95b121611f45ee210d280baabb8/src/manage_nsfs/manage_nsfs_constants.js#L4-L40

Reply:
I think `validate_type_and_action` function doing the same thing so calling new function after that. wdyt?

Reply:
I think you're right.